pywbem/cim_operations.py

```diff
@@ -587,6 +590,25 @@ def __init__(self, url, creds=None, default_namespace=DEFAULT_NAMESPACE,
             Initial enablement status for maintaining statistics about the
             WBEM operations executed via this connection. See the
             :ref:`Statistics` section for details.
+
+          enable_logging (:term:`string` or None):
+            *New in pywbem 0.12 as experimental
+            Enable logging of operation requests and/or xml requests and
+            responses and set the detail level for the logging.
```

Comment:
I suggest to change "operation requests and/or xml requests and responses" to "pywbem API calls and/or CIM-XML requests and responses".

Reply:
Done